doc/api/unicode.md

```diff
@@ -0,0 +1,55 @@
+# icu
```

Comment:
`unicode`? Also, it might be good to add this to `exports.builtinLibs` in `lib/internal/module.js`.